packages/boot/src/booters/booter-utils.ts

```diff
@@ -45,10 +45,13 @@ export function isClass(target: any): target is Constructor<any> {
 export function loadClassesFromFiles(files: string[]): Constructor<{}>[] {
   const classes: Array<Constructor<{}>> = [];
   for (const file of files) {
-    const data = require(file);
-    for (const cls of Object.values(data)) {
-      if (isClass(cls)) {
-        classes.push(cls);
+    const moduleObj = require(file);
+    // WORKAROUND: use `for in` instead of Object.values().
```

Comment:
Once the issue is fixed, I don't think we need to change this back ... it works the same so I would propose removing the comment.

Reply:
The purpose of the comment is recording keeping. When developers blame the git history, they will know why the change is made.